src/controllers/applicant.ts

```diff
@@ -59,3 +80,15 @@ export const listAllApplicants = async (req: Request, res: Response): Promise<vo
         res.status(500).send(error);
     }
 };
+
+export const findGrade = async (req: Request, res: Response): Promise<void> => {
+    try {
+        const grade = await ScreeningGrade.findOne({ applicant: req.body.id });
+        const applicant = await grade.populate('applicant').execPopulate();
+        const applicantJSON = applicant.toJSON();
+        const applicantName = applicantJSON.applicant.firstName + ' ' + applicantJSON.applicant.lastName;
+        res.status(201).send('The total grade for ' + applicantName + ' is: ' + grade.total);
```

Comment:
Sending the grade object here is probably better - we can deal with making more presentable strings on the front end